release-notes/opensearch-anomaly-detection.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,44 @@
+## Version 2.0.0.0-rc1 Release Notes
+
+Compatible with OpenSearch 2.0.0
+
+
+### Enhancements
+
+* changed usages of "master" to "clusterManager" in variable names ([#504](https://github.com/opensearch-project/anomaly-detection/pull/504))
+
+### Bug Fixes
+
+* Changed default description to empty string instead of null ([#438](https://github.com/opensearch-project/anomaly-detection/pull/438))
+* Fixed ADTaskProfile toXContent bug and added to .gitignore ([#447](https://github.com/opensearch-project/anomaly-detection/pull/447))
+* Fix restart HCAD detector bug ([#460](https://github.com/opensearch-project/anomaly-detection/pull/460))
+* Check if indices exist in the presence of empty search results ([#495](https://github.com/opensearch-project/anomaly-detection/pull/495))
+
+### Infrastructure
+
+* Reduced jacoco exclusions and added more tests ([#446](https://github.com/opensearch-project/anomaly-detection/pull/446))
+* Remove oss flavor ([#449](https://github.com/opensearch-project/anomaly-detection/pull/449))
+* Add auto labeler workflow ([#455](https://github.com/opensearch-project/anomaly-detection/pull/455))
+* Gradle 7 and Opensearch 2.0 upgrade ([#464](https://github.com/opensearch-project/anomaly-detection/pull/464))
+* Add support for -Dbuild.version_qualifier ([#468](https://github.com/opensearch-project/anomaly-detection/pull/468))
+* change default version to 2.0.0-alpha1-SNAPSHOT ([#472](https://github.com/opensearch-project/anomaly-detection/pull/472))
+* revert alpha1 default version ([#473](https://github.com/opensearch-project/anomaly-detection/pull/473))
```

Comment:
nit: commits like these can usually be ignored in release notes. Not a blocker though

Reply:
+1 on some of the version qualifier stuff

Reply:
Not sure if we should include only the important PRs in release notes. But when we release ml-commons, someone told me we should include all PRs. Anyone know some formal guideline @ohltyler @amitgalitz @peterzhuamazon ?

Reply:
I am not aware of any guidelines but have been told previously if a commit is something like reverting to alpha1 and then anyway we have a commit to change default version. We can remove one of those. I do think it makes sense to have a guideline however. I see why keeping them all makes sense also but it can get kinda messy if we are just commit really small one line changes to our infra or they are PR that we reverted with a later PR cause of a bug or mistake making the UX of reading the changing log messy but I don't have a strong say on this.